core/src/components/content/content.scss

```diff
@@ -54,10 +54,6 @@
   color: current-color(contrast);
 }
 
-:host(.outer-content) {
```

Comment:
This class is applied in tests, but the applicable CSS is also applied in those tests

Reply:
It looks like the work to change the tests was done here but not removing the actual style from content: https://github.com/ionic-team/ionic-framework/pull/16589

It was never documented as a breaking change here though: https://github.com/ionic-team/ionic-framework/blob/main/BREAKING_ARCHIVE/v4.md

Something to keep in mind as developers could be still using this since it continued to work.

Reply:
Should I target v8 then? Or maybe v7.6?

Reply:
Possibly v8 if we want to be extra cautious. I don't know who might still be using it.

Reply:
I would be good with targeting v8 and documenting as a breaking change. Since this is just removing a few lines of code, I don't think there's a strong reason to release sooner.

Reply:
Ok all set. This code now targets v8